src/components/Header/Header.js

```diff
@@ -103,11 +99,12 @@ Header.defaultProps = {
   menuProps: {
     onSignOut: () => {},
     isAuthenticated: true,
-    signOutPath: "/",
-    dashboardPath: "/",
-    securityPath: "/",
-    profilePath: "/",
-    craPath: "/",
+    menuList: [
+      { key: "dashKey", value: "My dashboard", path: "/" },
+      { key: "securityKey", value: "Security Settings", path: "/" },
+      { key: "profileKey", value: "Profile", path: "/" },
+      { key: "outKey", value: "Sign out", path: "/" },
+    ],
```

Comment:
Is the CRA path missing?

Reply:
These values will be provided by the application adding the header through props